readme.md

```diff
@@ -163,6 +163,41 @@ You can set `data-image` and `data-description` attributes to set up an image an
 <div class="vkontakte" data-image="https://placekitten.com/200/400" data-description="Check this out">Share</div>
 ```
 
+### Reinitialize configuration on change data attributes
+
+If you need to dynamically change the configuration of the widget for sharing, you can update the date attributes and call the configuration update logic on the *Likely* instance using the `updateOptionFromDataSet` method.
+
+
+1. Likely widget in html template
+
+```html
+<div class="likely" data-url="https://google.com">
+    <div class="twitter">Tweet</div>
+    <div class="facebook">Share</div>
+    <div class="vkontakte">Share</div>
+    <div class="odnoklassniki">Like</div>
+    <div class="pinterest">Pin</div>
+    <div class="telegram">Send</div>
+    <div class="linkedin">Share</div>
+    <div class="whatsapp">Send</div>
+</div>
+```
+
+3. Some logic for update data attributes
+
+2. Logic for update configuration on instance *Likely*
```

Comment:
It's weird but the list is rendered as 1, 3, 4 (check out the preview on GitHub) 🤔 
I don't get the second item, "some logic to update attributes", could you elaborate, pls?

Reply:
I will rewrite the example so that it displays normally.